packages/rocketchat-autotranslate/client/lib/autotranslate.js

```diff
@@ -41,11 +42,15 @@ RocketChat.AutoTranslate = {
 		Meteor.call('autoTranslate.getSupportedLanguages', 'en', (err, languages) => {
 			this.supportedLanguages = languages || [];
 		});
-
 		Tracker.autorun(() => {
 			if (RocketChat.settings.get('AutoTranslate_Enabled') && RocketChat.authz.hasAtLeastOnePermission(['auto-translate'])) {
 				RocketChat.callbacks.add('renderMessage', (message) => {
-					const subscription = RocketChat.models.Subscriptions.findOne({ rid: message.rid }, { fields: { autoTranslate: 1, autoTranslateLanguage: 1 } });
+					const subscription = RocketChat.models.Subscriptions.findOne({rid: message.rid}, {
```

Comment:
Not chaning this reduces the diff and makes it easier to merge. Though it looks less beautiful, kindly minimize the footprint